packages/gatsby-transformer-mdx-introspection/src/generate-component-node-object.js

```diff
@@ -4,6 +4,14 @@ const generateComponentNodeObject = jsxString => {
   const jsxAsDom = parse5.parseFragment(jsxString, {
     scriptingEnabled: false,
   });
+
+  // In case we don't get any childNodes, we return early.
+  // Example:
+  //   { nodeName: '#document-fragment', childNodes: [] }
+  if (jsxAsDom.childNodes.length === 0) {
```

Comment:
This happens for example for `</SubTitle>`. For now, we simply return and don't create any node.

cc @nkuehn @davifantasia